rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
```

Comment:
- [ ] I select this part because I can't select the concerned part, these changes should be done in person.rb file 🙃 
  - Getters and Setters for `@rentals` should be present,
  - `@rentals` should be initialized,
  - A method for adding rentals should be present,
  - The person itself should be sent as a parameter to create the new rental achieving the `has-many` association